includes/admin/class-wcs-admin-system-status.php

```diff
@@ -32,8 +31,9 @@ public static function init() {
 	 * @since 1.0.0 - Migrated from WooCommerce Subscriptions v2.3.0
 	 */
 	public static function render_system_status_items() {
-
-		$store_data = $subscriptions_data = $subscriptions_by_payment_gateway_data = $payment_gateway_data = array();
```

Comment:
The `$subscriptions_by_payment_gateway_data` variable was unused, so I've removed it.

Reply:
Oh wow, we very rarely use this approach nowadays but good pick up on the incorrect variable name being created here and never used.